src/background/in-dapp-notifications.ts

```diff
@@ -0,0 +1,84 @@
+import browser from 'webextension-polyfill';
+import { networksStore } from 'src/modules/networks/networks-store.background';
+import type { ChainId } from 'src/modules/ethereum/transactions/ChainId';
+import type { Chain } from 'src/modules/networks/Chain';
+import type { InDappNotification } from 'src/shared/types/InDappNotification';
+import { emitter } from './events';
+
+declare global {
+  interface Window {
+    showNotification(notification: InDappNotification): void;
+  }
+}
+
+async function getActiveTabIdWithOrigin(origin: string) {
```

Comment:
What does it mean "with origin"?
maybe `getActiveTabIdByOrigin`?

Reply:
natural language / direct translation issues :)